src/s_print.c

```diff
@@ -146,7 +146,7 @@ static void dologpost(const void *object, const int level, const char *s)
     }
 }
 
-void logpost(const void *object, const int level, const char *fmt, ...)
+void logpost(const void *object, int level, const char *fmt, ...)
```

Comment:
i know that the `const` qualifier for the level is of little actual use, but is there a specific reason to remove it?

Reply:
because there is no reason for it to be there in the first place :-)